components/salesforce_rest_api/common/sobjects/task.mjs

```diff
@@ -0,0 +1,150 @@
+    options: [
+      "None",
+      "After due date",
+      "After the task is closed",
+      "(Task closed)",
+    ],
+  },
+  RecurrenceStartDateOnly: {
+    type: "string",
+    label: "RecurrenceStartDateOnly",
+    description: "The date when the recurring task begins. Must be a date and time before RecurrenceEndDateOnly.",
+  },
+  RecurrenceTimeZoneSidKey: {
+    type: "string",
+    label: "RecurrenceTimeZoneSidKey",
+    description: "The time zone associated with the recurring task. For example, UTC-8:00 for Pacific Standard Time.",
+  },
+  RecurrenceType: {
+    type: "string",
+    label: "RecurrenceType",
+    description: "Indicates how often the task repeats. For example, daily, weekly, or every nth month (where nth is defined in RecurrenceInstance).",
+  },
+  ReminderDateTime: {
+    type: "string",
+    label: "ReminderDateTime",
+    description: "Represents the time when the reminder is scheduled to fire, if IsReminderSet is set to true. If IsReminderSet is set to false, then the user may have deselected the reminder checkbox in the Salesforce user interface, or the reminder has already fired at the time indicated by the value.",
+  },
+  Subject: {
+    type: "string",
+    label: "Subject",
+    description: "The subject line of the task, such as Call or Send Quote. Limit: 255 characters.",
+  },
+  TaskSubtype: {
+    type: "string",
+    label: "TaskSubtype",
+    description: "Provides standard subtypes to facilitate creating and searching for specific task subtypes. This field isn't updateable. TaskSubtype values: Task Email List Email Cadence Call Note The Cadence subtype is an internal value used by High Velocity Sales, and can't be set manually.",
+    options: [
+      "Task",
+      "Email",
+      "List Email",
+      "Cadence",
+      "Call",
+    ],
+  },
+  TaskWhoIds: {
+    type: "any",
+    label: "TaskWhoIds",
+    description: "A string array of contact or lead IDs related to this task. This JunctionIdList field is linked to the TaskWhoRelations child relationship. TaskWhoIds is only available when the shared activities setting is enabled. The first contact or lead ID in the list becomes the primary WhoId if you don't specify a primary WhoId. If you set the EventWhoIds field to null, all entries in the list are deleted and the value of WhoId is added as the first entry. Warning Adding a JunctionIdList field name to the fieldsToNull property deletes all related junction records. This action can't be undone.",
+  },
+  Type: {
+    type: "string",
+    label: "Type",
+    description: "The type of task, such as Call or Meeting.",
+  },
+  WhatId: {
+    type: "string",
+    label: "WhatId",
+    description: "The WhatId represents nonhuman objects such as accounts, opportunities, campaigns, cases, or custom objects. WhatIds are polymorphic. Polymorphic means a WhatId is equivalent to the ID of a related object.",
+  },
+  WhoId: {
```

Comment:
Apply the `camelCase` pattern for the prop key.

You can do this for the other keys too.